CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/zoom/ZoomOutPlugin.java

```diff
@@ -0,0 +1,67 @@
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.graph.interaction.plugins.zoom;
+
+import au.gov.asd.tac.constellation.graph.Graph;
+import au.gov.asd.tac.constellation.graph.GraphWriteMethods;
+import au.gov.asd.tac.constellation.graph.interaction.animation.Animation;
+import au.gov.asd.tac.constellation.graph.interaction.animation.PanAnimation;
+import au.gov.asd.tac.constellation.graph.manager.GraphManager;
+import au.gov.asd.tac.constellation.graph.visual.utilities.VisualGraphUtilities;
+import au.gov.asd.tac.constellation.plugins.Plugin;
+import au.gov.asd.tac.constellation.plugins.PluginException;
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import au.gov.asd.tac.constellation.plugins.PluginInteraction;
+import au.gov.asd.tac.constellation.plugins.PluginType;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameters;
+import au.gov.asd.tac.constellation.plugins.templates.PluginTags;
+import au.gov.asd.tac.constellation.plugins.templates.SimpleEditPlugin;
+import au.gov.asd.tac.constellation.utilities.camera.Camera;
+import au.gov.asd.tac.constellation.utilities.camera.CameraUtilities;
+import au.gov.asd.tac.constellation.utilities.graphics.Vector3f;
+import org.openide.util.NbBundle;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ *
+ * @author Quasar985
+ */
+@ServiceProvider(service = Plugin.class)
+@NbBundle.Messages("ZoomOutPlugin=Zoom Out")
+@PluginInfo(minLogInterval = 5000, pluginType = PluginType.VIEW, tags = {PluginTags.VIEW})
+public final class ZoomOutPlugin extends SimpleEditPlugin {
+
+    final Vector3f ZOOM_DIRECTION = new Vector3f(0, 0, -1);
+    final int ZOOM_AMOUNT = 10;
+
+    @Override
+    public void edit(final GraphWriteMethods graph, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException, PluginException {
+        final Camera oldCamera = VisualGraphUtilities.getCamera(graph);
+        final Camera camera = new Camera(oldCamera);
+
+        CameraUtilities.zoom(camera, ZOOM_AMOUNT, ZOOM_DIRECTION, 0F);
+
+        final Graph activeGraph = GraphManager.getDefault().getActiveGraph();
+        if (activeGraph != null && activeGraph.getId().equals(graph.getId())) {
+            // Only do the camera animation if the edited graph is currently active
+            Animation.startAnimation(new PanAnimation("Zoom to Selection", oldCamera, camera, true));
```

Comment:
Running this as an animation looks nice when users zoom with a gap between zoom requests. However, when zooming quicky. (i.e ctrl + alt + hold up arrow) the experiences a little jarring with pan animations stopping and starting. I wonder if simply writing to the graph and not doing an animation would be nicer? Single requests to zoom would be more jarring but might be a lesser evil. (Same goes for ZoonInPlugin)